app/mailers/due_soon_and_overdue_defects_mailer.rb

```diff
@@ -1,5 +1,11 @@
 class DueSoonAndOverdueDefectsMailer < ApplicationMailer
   def notify(defect_ids)
-    @defects = Defect.find(defect_ids)
+    @defects = Defect.find(defect_ids).map { |defect| DefectPresenter.new(defect) }
+    view_mail(
+      NOTIFY_DAILY_DUE_SOON_TEMPLATE,
+      to: NBT_GROUP_EMAIL,
+      subject: I18n.t('email.defects.due_soon_and_overdue.subject'),
+      template_name: 'notify',
```

Comment:
Do we need to specify this, or should Rails just be able to work this out?

Reply:
Ah, yeah probably not, this was a refactoring step I missed from when there were a couple of template names.

Reply:
With the other refactorings going on, this is now required since the call site means Rails by default looks for another template.